unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/AvatarSlot/AvatarSlotComponentView.cs

```diff
@@ -89,33 +109,38 @@ public override void RefreshControl()
         public void SetHideList(string[] hideList) =>
             model.hidesList = hideList;
 
+        public void SetOverrideHide(bool isOverridden)
+        {
+            overriddenHide.gameObject.SetActive(isOverridden);
+            normalHide.gameObject.SetActive(!isOverridden);
+        }
+
         public void SetIsHidden(bool isHidden, string hiddenBy)
         {
             model.isHidden = isHidden;
-            model.hiddenBy = hiddenBy;
-            hiddenSlot.SetActive(isHidden);
 
             if (isHidden)
+                hiddenByList.Add(hiddenBy);
+            else
+                hiddenByList.Remove(hiddenBy);
+
+            List<string> sortedList = hiddenByList.OrderBy(x => WearableItem.CATEGORIES_PRIORITY.IndexOf(x)).ToList();
```

Comment:
what if `hiddenByList` is a `SortedList<int, string>`? we could avoid the allocation of this list